build.gradle

```diff
@@ -28,5 +28,6 @@ dependencies {
     implementation('org.springframework.boot:spring-boot-starter-data-jpa')
     implementation('org.springframework.boot:spring-boot-starter-web')
     runtimeOnly('com.h2database:h2')
+    runtimeOnly 'mysql:mysql-connector-java'
```

Comment:
jeśli dorzuciłes driver do mysql, to driver do h2 zmien na testRuntime

Reply:
dzieki temu h2 bedzie uzywana tylko w testach, a 'normalnym dzialaniu' aplikacji bedzie mysql